sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SetupFlowListener.kt

```diff
@@ -17,5 +17,9 @@
 package com.duckduckgo.sync.impl.ui.setup
 
 interface SetupFlowListener {
-    fun launchFinishSetupFlow()
+    fun launchRecoveryCodeFlow()
```

Comment:
I think previously `Flow` made more sense than now where we are just showing screens...can we just rename `xxxFlow()` by `xxxxScreen()`?